base/src/com/google/idea/blaze/base/ui/problems/ImportIssueResolver.java

```diff
@@ -0,0 +1,69 @@
+package com.google.idea.blaze.base.ui.problems;
+
+import com.google.idea.blaze.base.scope.output.IssueOutput;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.fileEditor.FileDocumentManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.Computable;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiManager;
+
+import java.util.regex.Pattern;
+
+public class ImportIssueResolver {
+
+    private static final String NEW_LINE = "\n";
+    private static String missingImportRegEx = "object .* is not a member of package .*";
+    private static String wildCardOrStaticRegEx = "package .* does not exist";
+    private static String javaStaticImport = "static import only from classes and interfaces";
```

Comment:
Shouldn't problems like this already be annotated in the IDE?

Reply:
they are not currently